tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -154,10 +154,15 @@ impl Default for Format<Full, SystemTime> {
 impl<T> Format<Json, T> {
     /// Use the full JSON format with event metadata flattened.
     ///
+    /// # Example Output
+    ///
+    /// ```ignore,json
+    /// {"timestamp":"Feb 20 11:28:15.096","level":"INFO","target":"mycrate", "message":"some message", "key": "value"}
```

Comment:
take it or leave it: it might be nice to run this through `jq` so so it's not all on one line...of course, then it won't be the actual output, and we'd have to note that...I don't really care.

Reply:
I think it looks fine in the rendered docs and represents the actual output.